src/lv_widgets/lv_arc.c

```diff
@@ -379,23 +380,13 @@ void lv_arc_set_type(lv_obj_t * arc, lv_arc_type_t type)
 }
 
 /**
- * Set a new value on the arc
+ * Used internally to update arc angles after a value change
  * @param arc pointer to a arc object
  * @param value new value
  */
-void lv_arc_set_value(lv_obj_t * arc, int16_t value)
+void value_update(lv_obj_t * arc)
```

Comment:
As it's an internal function please make it `static` and move to the bottom of the file.

Reply:
will do

Reply:
moved to header and made it inline static. Technically not a "private" function anymore, but for the sake of consistency I get why you wanted the change.

Reply:
I think @kisvegabor wanted it `static` (not inline) in the C file.

Reply:
ah gotcha, will do

Reply:
done @embeddedt ty for clarification